Search via POST using multiple comparison operators fails

**Describe the bug**
When performing a FHIR search **via `POST`** multiple comparison operators targeting a single property fail with `BadRequest` response. The **same request** sent via `GET` succeeds.

**To Reproduce**
Steps to reproduce the behavior:

1. Create a `FhirClient` (using `Hl7.Fhir.R4` library) targeting a Spark server (e.g., the `incendi.no` reference server).
2. Create `SearchParams` which include two comparison operators targeting the same resource property. Known affected resource: `DocumentReference`; known affected search property: `period`. See example code below for known reproducible arrangement.
3. Send the request using `POST`. (I.e., `fhirClient.SearchUsingPostAsync<>()`)
    * The failure presents as an exception: `FhirOperationException: Operation was unsuccessful because of a client error (BadRequest). OperationOutcome: Overall result: FAILURE (2 errors and 0 warnings)
[ERROR] (no details)(further diagnostics: Could not parse parameter [(period, ge1983-08-01T23:49:57.0000000+07:00,lt1983-10-03T22:57:57.0000000+07:00)] for reason [Multiple values cannot be used in combination with a comparison operator].)`

To reproduce failure:

```csharp
var searchParams = new SearchParams();
searchParams.Add("period", $"ge1983-08-01T23:49:57.0000000+07:00");
searchParams.Add("period", $"lt1983-10-03T22:57:57.0000000+07:00");
var documents = await fhirClient.SearchUsingPostAsync<DocumentReference>(searchParams);
```

To reproduce a _successful_ identical search (using `GET`):

```csharp
var searchParams = new SearchParams();
searchParams.Add("period", $"ge1983-08-01T23:49:57.0000000+07:00");
searchParams.Add("period", $"lt1983-10-03T22:57:57.0000000+07:00");
var documents = await fhirClient.SearchAsync<DocumentReference>(searchParams);
```

**Expected behavior**
Expected that a successful `OK` response occurs (such as observed when following the example shown using `GET`, above).

**Spark version**
  - Version: `2.0.0`

**Operating system + Database**
  - OS: Linux (Debian); Windows 11
  - Database: MongoDB

**Container service / Cloud infrastructure:**
  - Container service: Docker Desktop; AWS ECS
  - Cloud provider: AWS
  - Cloud infrastructure: Containerized
  - Database as a service: AWS DocumentDB

--- a/src/Spark.Engine/Extensions/HttpHeadersExtensions.cs
+++ b/src/Spark.Engine/Extensions/HttpHeadersExtensions.cs
@@ -71,12 +71,10 @@ namespace Spark.Engine.Extensions
         public static SearchParams GetSearchParamsFromBody(this HttpRequest request)
         {
             var list = new List<Tuple<string, string>>();
-
             foreach (var parameter in request.Form)
             {
-                list.Add(new Tuple<string, string>(parameter.Key, parameter.Value));
+                list.AddRange(parameter.Value.Select(value => new Tuple<string, string>(parameter.Key, value)));
             }
-
             return request.GetSearchParams().AddAll(list);
         }
 

